build/Targets/Settings.props

```diff
@@ -124,6 +124,7 @@
     <When Condition="'$(MSBuildProjectExtension)' == '.vbproj' OR '$(Language)' == 'VB' OR '$(ProjectLanguage)' == 'VB'">
       <PropertyGroup>
         <ProjectLanguage>VB</ProjectLanguage>
+        <LangVersion>Latest</LangVersion>
```

Comment:
I think this is just creating busy work for us.  We're always in the habit of using the latest language features hte moment they are available in master.  Using the hard coded language version here isn't going to save us anything.  It's just going to lead to emails complaining about how they can't use C# latest in master and a subsequent change to up this version.